python_ta/cfg/visitor.py

```diff
@@ -199,3 +199,41 @@ def _visit_jump(self, node: Union[astroid.Break, astroid.Continue, astroid.Retur
                               f' {"function" if isinstance(node, astroid.Return) else "loop"}')
         unreachable_block = self._current_cfg.create_block()
         self._current_block = unreachable_block
+
+    def visit_tryexcept(self, node: astroid.TryExcept) -> None:
+        self._current_block = self._current_cfg.create_block(self._current_block)
+        node.cfg_block = self._current_block
+
+        for child in node.body:
+            child.accept(self)
+        end_body = self._current_block
+
+        end_block = self._current_cfg.create_block()
+
+        handlers = []
+        for handler in node.handlers:
+            h = self._current_cfg.create_block()
+            self._current_block = h
+            handler.cfg_block = h
+            for child in handler.body:
+                child.accept(self)
+            end_handler = self._current_block
+            self._current_cfg.link_or_merge(end_handler, end_block)
+            handlers.append(h)
+
+        after_body = handlers
```

Comment:
In general, prefer not creating aliases. You can name the original variable (line 213) `after_body` and stick with that throughout.